src/lein_monolith/task/graph.clj

```diff
@@ -9,6 +9,11 @@
 
 (def image-name "project-hierarchy.png")
 
+(defn cluster->descriptor
+  [monolith-root subdirectory]
+  (when-not (= monolith-root subdirectory)
```

Comment:
Style: I have a minor preference for `(when (not= ...` here. Please preserve the two-line spacing between top-level vars as well.